lib/image.js

```diff
@@ -46,21 +43,30 @@ Object.defineProperty(Image.prototype, 'src', {
             throw err
           }
         }
-
-        if (!get) get = require('simple-get')
-
-        get.concat({
-          url: val,
-          headers: { 'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36' }
-        }, (err, res, data) => {
-          if (err) return onerror(err)
-
+        util
+        .callbackify(
+          fetch(val, {
+            method: 'POST',
```

Comment:
I'm not sure if the changes I made are correct.

simple-get.concat method type is GET.

Reply:
=> https://github.com/Automattic/node-canvas/pull/2309/commits/1eb26cf66bf2b8c3a35c7e02a2da0dd230248e24